zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/CassandraSpanConsumer.java

```diff
@@ -14,7 +14,6 @@
 package zipkin2.storage.cassandra.v1;
 
 import com.datastax.driver.core.Session;
-import com.google.common.cache.CacheBuilderSpec;
```

Comment:
changes like this, just strangle guava usages to be replaced by DelayLimiter into one place